keylime.conf

```diff
@@ -665,6 +665,15 @@ transparency_log_sign_algo = sha256
 # will mean no signing should be done.
 signed_attributes = ek_tpm,aik_tpm,ekcert
 
+# What TPM-based identity is allowed to be used to register agents.
+# The options "default" and "iak_idevid" will only allow registration with IAK and IDevID if python cryptography is version 38.0.0 or higher.
+# The following options are accepted:
+# "default": either an EK or IAK and IDevID may be used. In the case that cryptography version is <38.0.0 only EK will be used
+# "ek_cert_or_iak_idevid": this is equivalent to default
+# "ek_cert": only allow agents to use an EK to register
+# "iak_idevid": only allow agents with an IAK and IDevID to register
+tpm_identity = default
```

Comment:
Should this be in a j2 file as well?

Reply:
Nice catch! Yes, this should be in the `registrar.j2`. I see it is already in the `mapping.json`, but if it is not in the template, it will not be added to the `registrar.conf` in the output

Reply:
Who is now using the j2 in 2.1 directory?

Reply:
This PR is introducing the 2.1 version for the configuration templates.

Reply:
@Isaac-Matthews if you fix this last problem, I can include it on this month's release

Reply:
> This PR is introducing the 2.1 version for the configuration templates.

So convert_config will now pick up the 2.1 templates rather than the old 2.0 ones?

Reply:
Ah sorry looks like I only updated the mappings when we changed config option. This is now updated with the old option removed.

Reply:
Now that we have 2.1 templates is anyone still going to use the 2.0 templates?

Reply:
The upgrade script applies all mappings in sequence and then outputs the configuration files using the latest templates, but can also provide `--target` to stop at an intermediary version. So, the version `2.0` can still be used.